src/pages/location-message/location-message.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
```

Comment:
I don't use Sublime. if you want a specific lint config, please add it to the tslint.json